lib/kubernetes-deploy/render_task.rb

```diff
@@ -3,18 +3,16 @@
 
 require 'kubernetes-deploy/common'
 require 'kubernetes-deploy/renderer'
+require 'kubernetes-deploy/template_sets'
 
 module KubernetesDeploy
   class RenderTask
-    def initialize(logger: nil, current_sha:, template_dir:, bindings:)
+    def initialize(logger: nil, current_sha:, template_dir: nil, template_paths: [], bindings:)
```

Comment:
Now that we can make breaking changes should we start retiring the deprecated `template_dir` argument in favour of exclusively using `template_paths`

Reply:
I'd like to do handle this in a follow-up pr. I think this one is big enough as is.